infra/build/blog/deploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+
+docker build -t oss-fuzz-blog .
+
+rm -rf site
+mkdir -p site
+
+docker build -t oss-fuzz-blog .
+docker run --rm -ti -v $(pwd)/site:/site oss-fuzz-blog sh -c 'cp -r /oss-fuzz-blog/page/public /site && chmod -R 777 /site'
```

Comment:
nit: Would `"$(pwd)"` be safer to avoid name splitting and globbing? It might not be applicable to this specific case.